apis/apps/v1alpha1/opsrequest_types.go

```diff
@@ -120,6 +129,21 @@ type ComponentOps struct {
 	ComponentName string `json:"componentName"`
 }
 
+type Switchover struct {
+	ComponentOps `json:",inline"`
+
+	// instanceName is used to specify the candidate primary or leader instanceName for switchover.
+	// If instanceName is set to "*", it means that no specific primary or leader is specified for the switchover,
+	// and the switchoverAction defined in clusterDefinition.componentDefs[x].switchoverSpec.withoutCandidate will be executed,
+	// It is required that clusterDefinition.componentDefs[x].switchoverSpec.withoutCandidate is not empty.
+	// If instanceName is set to a valid instanceName other than "*", it means that a specific candidate primary or leader is specified for the switchover.
+	// the value of instanceName can be obtained using `kbcli cluster list-instances`, any other value is invalid.
```

Comment:
what's the expected behavior for invalid instance name?

Reply:
> what's the expected behavior for invalid instance name?

Invalid InstanceName will be intercepted by kbcli and opsRequest validation, and the switchover will not be performed.
At the same time, ops controller runtime will not  perform a switchover task for an invalid InstanceName.